### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Spark Scala Application_Keyboard: Focus loss is observed after tabbing from “Project Location” file and “New” button.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3612]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.4.3; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS2.4.3

**Repro Steps:**
1.Download IntelliJ application from the URL:   https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA application.
3. Tab till Tools-->Azure-->Validate Spark Cluster SSL Certificate-->Disable and press enter
4. Tab till View Menu-->Tool Windows-->Azure Explorer and press enter
5. Right click on the SQL Server big data cluster-->Link SQL Server big data cluster and press enter
6. Enter all the required details to link the cluster
7. Tab till New-->Project and press enter
8. Select Azure Spark/HDInsight > Spark Project with Samples (Scala) and press Next.
9.Tab till “Project Location” field.
10.Verify whether focus is immediately pointing to the next controls when we press tab when focus is on “Project location” field and “New” button or not.

**Actual Result:**
Focus loss is observed after tabbing from “Project Location” file and “New” button.

**Expected Result:**
When focus is on “Project Location” filed and “New” button, when we press tab focus should immediately point to “Project SDK” and “Spark Version” fields respectively.

**User Impact:**
Keyboard users will face difficulty in knowing where the focus is pointing if Focus is not visible for 1 tab before going to “Project SDK” and “Spark Version” fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_Focus order issue in spark scala application page.pptx](https://github.com/microsoft/azure-tools-for-java/files/3646507/Bug_Focus.order.issue.in.spark.scala.application.page.pptx)

**Comment by t-rufang** [comment]
Note: 2 bugs here:
1. When focus is at `Project location` text field, tab once, focus is invisible.
2. When focus is at `New...` button, tab once, focus is invisible.
![image](https://user-images.githubusercontent.com/32627233/65660338-06150080-e061-11e9-904f-08f5d0de3296.png)

**Comment by t-rufang** [comment]
@v-lasiva For the second issue, it's fixed in #3678. Please help to verify the fix.

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. When focus is at `Project location` field, tab once , focus is invisible. Keyboard users will face difficulty in knowing where the focus is pointing now.
- Quality of experience: management. Tab one mote time focus will be visible and at `Project SDK` combo box.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
As per discussion with SME, Closing the issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749712
